README.md

```diff
@@ -73,28 +76,28 @@ if ( function_exists( 'vip_geo_get_country_code' ) && 'GB' == vip_geo_get_countr
 
 ## Caching for sites using geo targeting
 
-When you enable the VIP Go Geo Uniques plugin, your page cache is varied by country code. This means that, for each URL requested, we hold a separate copy of the page (i.e. the HTTP response) for each country in our page cache. When you update a post or page, all cached copies for all country codes are automatically invalidated. When you purge the cache for your whole site, the whole cache is still invalidated.
+When you enable the VIP Go Geo Uniques plugin, your page cache is varied by country code. This means that for each URL requested, we hold a separate copy of the page (i.e. the HTTP response) for each country in our page cache. When you update a post or page, all cached copies for all country codes are automatically invalidated. When you purge the cache for your whole site, the whole cache is still invalidated.
 
 ## Geo targeting and reverse proxies
 
-Geo targeting on VIP Go is not reliable if your site uses a reverse proxy.
+Geo targeting on VIP is not reliable if your site uses a reverse proxy.
 
-This is because when a request comes via a reverse proxy, the remote IP address is that of the reverse proxy. The load balancers on VIP Go will assign the country code to the reverse proxy IP, and not the IP address of the end user; sometimes this may by chance be the right country for the end user, but many times it will not be.
+This is because when a request comes via a reverse proxy, the remote IP address is that of the reverse proxy. The load balancers on VIP will assign the country code to the reverse proxy IP, and not the IP address of the end user. It is possible for the correct country to be served to the end user by chance, but in most cases times it will not be correct.
```

Comment:
```suggestion
This is because when a request comes via a reverse proxy, the remote IP address is that of the reverse proxy. The load balancers on VIP will assign the country code to the reverse proxy IP and not the end user's IP address. The correct country can be served to the end user by chance, but in most cases, it will not be correct.
```